scripts/testenv/testenvSettings.sh

```diff
@@ -1,18 +1,25 @@
 #!/usr/bin/env bash
 set +x
+set -e
 
-if [[ $USE_TESTENV_PROPERTIES == true ]]; then
-    testenv_file="./testenv/testenv.properties"
-    if [[ "$PLATFORM" == *"zos"* ]]; then
-        testenv_file="./testenv/testenv_zos.properties"
-    fi
-    if [[ "$PLATFORM" == *"arm"* ]] && [[ "$JDK_VERSION" == "8" ]]; then
-        testenv_file="./testenv/testenv_arm32.properties"
-    fi
+if [ $USE_TESTENV_PROPERTIES = true ]; then
+    case "$PLATFORM" in
+        *zos*)
+            testenv_file="./testenv/testenv_zos.properties"
+            ;;
+        *arm*)
+            if [ "$JDK_VERSION" = "8" ]; then
+                testenv_file="./testenv/testenv_arm32.properties"
```

Comment:
Is it expected that testenv_file won't be set for other versions than 8 of arm?

Reply:
I think so. I just changed the code to use ' case` instead.

Reply:
The original code used testenv.properties for arm non-8, but this code doesn't set testenv_file at all in this case. I guess non-8 arm doesn't exist, so it doesn't matter?

Reply:
yes, you are correct. I missed the case. PR is updated.